src/components/Video/VideoRecording.vue

```diff
@@ -215,8 +277,52 @@ export default {
     deleteTag(tagId) {
       this.$store.dispatch("Video/DELETE_TAG", tagId);
     },
-    trackSelected(track) {
-      this.selectedTrack = { trackIndex: track };
+    changedTrackTag(trackTag) {
+      this.recentlyAddedTrackTag = trackTag;
+      setTimeout(() => {
+        this.recentlyAddedTrackTag = null;
+      }, 2000);
+    },
+    async trackSelected(track) {
+      const selectedTrack = {
+        ...track,
+      };
+      const targetTrack = this.tracks[track.trackIndex];
+      if (track.gotoStart) {
+        selectedTrack.start_s = Math.max(
+          0,
+          targetTrack.data.start_s - this.timespanAdjustment
```

Comment:
Hmm might have been better to have the timespan Adjustment in one place (not this time though since we don't want to retest)